docker/ssl/Makefile

```diff
@@ -11,29 +11,29 @@ all: clean_certs ca server_crt
 # CA name and key is "ca".
 ca:
 	openssl req -newkey rsa:2048 -x509 -nodes -sha512 -days 1095 \
-				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=localhost/O=Mainflux/OU=IoT/emailAddress=info@mainflux.com"
+				-keyout $(CRT_LOCATION)/ca.key -out $(CRT_LOCATION)/ca.crt -subj "/CN=$(CN)/O=$(O)/OU=$(OU)/emailAddress=$(EA)"
 
 # Server cert and key name is "mainflux-server".
 server_cert:
-	# Create mainflux server key and CSR.
+# Create mainflux server key and CSR.
```

Comment:
Donot move these comments to the beginning of the line - they belong to indented code.

Reply:
Besides, it will be treated as a command by shell and it will be echoed as a comment.

Reply:
`Blank lines and lines of just comments may appear among the recipe lines; they are ignored.` - they should be completely ignored.

Did you try and verify if the shell echoes these in the case they are indented and not in the case they are not?